balance_controller:
  virtual_model_controller:
    heading:
      kp: 300
      kd: 40
      kff: 1
    lateral:
      kp: 800
      kd: 100
      kff: 1
    vertical:
      kp: 800
      kd: 100
      kff: 1
    roll:
      kp: 250
      kd: 100
      kff: 0.2
    pitch:
      kp: 400
      kd: 80
      kff: 0.2
    yaw:
      kp: 200
      kd: 50
      kff: 1
    gravity_compensation_percentage: 1.0
  contact_force_distribution:
    weights:
      force:
        heading: 1
        lateral: 1
        vertical: 1
      torque:
        roll: 10
        pitch: 10
        yaw: 5
      regularizer:
        value: 0.0001
    constraints:
      friction_coefficient: 0.6
      minimal_normal_force: 10
single_leg_controller:
  x_direction:
    kp: 300
    kd: 20
  y_direction:
    kp: 300
    kd: 20
  z_direction:
    kp: 800
    kd: 80
